.github/workflows/build-samples.yml

```diff
@@ -31,6 +33,49 @@ jobs:
         run: |
           ./apko build ./examples/nginx.yaml nginx:build /tmp/nginx-${{ matrix.arch }}.tar --debug --arch ${{ matrix.arch }}
 
+      - name: Check SBOM Conformance
+        run: |
+          set -euxo pipefail
+          for f in *.spdx.json; do
```

Comment:
Here and elsewhere it would be good if the check failed if we were to stop emitting SBOMs here.  I'm not sure what this would do if the wildcard matches 0 things...